docs/docsite/rst/reference_appendices/general_precedence.rst

```diff
@@ -167,7 +167,7 @@ This category only applies to things that take direct options, generally modules
 
 Outside of task actions, the most recognizable 'direct assignments' are with lookup, filter and test plugins:
 
-.. code::
+.. code:: text
```

Comment:
Are these pieces of Jinja?

Would
```suggestion
.. code:: yaml+jinja
```
handle this better?

Reply:
This is raw Jinja, not a Jinja template. Both `yaml+jinja` and `jinja` formatters are for content with Jinja templates, which this does not contain.